Sources_v3/WebSocketClient/EventMiddlewares/ChannelMemberTypingStateUpdaterMiddleware_Tests.swift

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatClient
+import XCTest
+
+final class ChannelMemberTypingStateUpdaterMiddleware_Tests: XCTestCase {
+    var database: DatabaseContainerMock!
+    var middleware: ChannelMemberTypingStateUpdaterMiddleware<DefaultDataTypes>!
+    
+    // MARK: - Set up
+    
+    override func setUp() {
+        super.setUp()
+        
+        database = try! DatabaseContainerMock(kind: .inMemory)
+        middleware = ChannelMemberTypingStateUpdaterMiddleware(database: database)
+    }
+    
+    override func tearDown() {
+        database = nil
+        middleware = nil
```

Comment:
nit: You can use `AssertAsync.canBeReleased` and you will get a retain cycle tests for free ;-)

Reply:
The `middleware` is a struct so `AssertAsync.canBeReleased` is not applicable for it.
Added the assert for `database` in 6471e38f6d14e5f04a41bf5d168785f7b5016dd3
Thanks 👍